setup.py

```diff
@@ -22,7 +22,7 @@
         "Operating System :: OS Independent",
     ],
     install_requires=[
-        "boututils~=0.1.4",
+        "boututils~=0.1.5",
```

Comment:
```suggestion
        "boututils~=0.1",
```

Maybe?

Reply:
It does need at least 0.1.5. It looks like `boututils~=0.1.5` means "any `boututils` version greater than or equal to 0.1.5 and less than 1" (although I'm not totally sure I'm reading the docs right). It might be better to put `boututils>=0.1.5` here, in the expectation that if we ever had a boututils-1.0.0 release, it would be backward compatible?